lib/dialects/postgres/schema/compiler.js

```diff
@@ -76,12 +76,20 @@ SchemaCompiler_PG.prototype.createSchemaIfNotExists = function(schemaName) {
   );
 };
 
-SchemaCompiler_PG.prototype.dropSchema = function(schemaName) {
-  this.pushQuery(`drop schema ${this.formatter.wrap(schemaName)}`);
+SchemaCompiler_PG.prototype.dropSchema = function(schemaName, cascade = false) {
+  this.pushQuery(
+    `drop schema ${this.formatter.wrap(schemaName)}${cascade && ' CASCADE'}`
```

Comment:
does this will work correct for cascade = false?
can you add unit test on this?